src/main/java/uk/gov/pay/directdebit/webhook/sandbox/resources/WebhookSandboxResource.java

```diff
@@ -47,12 +51,13 @@ public Response handleWebhook() {
     }
 
     private void processPayment(Payment payment) {
```

Comment:
Wonder whether this is better suited to being in the `WebhookSandboxService` than the resource.